### PR TITLE
FE: add a test to Tabs component

### PR DESCRIPTION
**Pull request by smonero**
### Description

Another test, this one for the Tabs component. This part of an wider effort to add more FE tests to Clutch.

### GitHub Issue

Fixes #

### CODE REVIEW

File: frontend/packages/core/src/tests/tab.test.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import { Tab, Tabs } from "../tab";
+
+describe("Tabs component aka TabGroup", () => {
```
Comment (dschaller) [inline frontend/packages/core/src/tests/tab.test.tsx:6]: I think it's ok to just call this Tabs component

File: frontend/packages/core/src/tests/tab.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  describe("basic rendering", () => {
```
Comment (dschaller) [inline frontend/packages/core/src/tests/tab.test.tsx:7]: nit:
```suggestion
  describe("with a value set", () => {
```

File: frontend/packages/core/src/tests/tab.test.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          <Tab label="mix" />
+        </Tabs>
+      );
+    });
+
+    it("renders", () => {
+      expect(component.find(Tabs)).toBeDefined();
+    });
+
+    it("renders 2 Tabs", () => {
```
Comment (dschaller) [inline frontend/packages/core/src/tests/tab.test.tsx:22]:
```suggestion
    it("renders children tabs", () => {
```

File: frontend/packages/core/src/tests/tab.test.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      expect(component.find(Tab)).toHaveLength(2);
+    });
+
+    it("has the second tab selected (index 1)", () => {
```
Comment (dschaller) [inline frontend/packages/core/src/tests/tab.test.tsx:26]:
```suggestion
    it("displays the tab from the specified value", () => {
```

File: frontend/packages/core/src/tests/tab.test.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      expect(component.find("TabContext").prop("value")).toBe("1");
```
Comment (dschaller) [inline frontend/packages/core/src/tests/tab.test.tsx:27]: Should we be expecting the mix tab to be selected and displayed?

Comment (smonero) [reply frontend/packages/core/src/tests/tab.test.tsx:27]: yes

Comment (smonero) [reply frontend/packages/core/src/tests/tab.test.tsx:27]: Are you saying I should add another test for that? I remember Josh suggested that we only have 1 `expect` per test to keep things organized

Comment (dschaller) [reply frontend/packages/core/src/tests/tab.test.tsx:27]: Yeah I would add a test that it's displayed as well

Comment (smonero) [reply frontend/packages/core/src/tests/tab.test.tsx:27]: here is the debug() output for the component:
```
<div style={{...}}>
@clutch-sh/core:       <TabContext value="1">
@clutch-sh/core:         <Styled(Component) variant={[undefined]} onChange={[Function: onChangeMiddleware]}>
@clutch-sh/core:           <Tab label="meow" value="0" />
@clutch-sh/core:           <Tab label="mix" value="1" />
@clutch-sh/core:         </Styled(Component)>
@clutch-sh/core:         <Styled(WithStyles(ForwardRef(TabPanel))) value="0" />
@clutch-sh/core:         <Styled(WithStyles(ForwardRef(TabPanel))) value="1" />
@clutch-sh/core:       </TabContext>
@clutch-sh/core:     </div>
```
as you can see it doesn't give an indication that "mix" tab is selected other than the tabContext . An alternative thing I could do is maybe use the react-test renderer like Josh does in the NPS thing instead. I'll ask Josh today about it.

**Review by jdslaugh — APPROVED** [review]

**Pull request merged**